src/Providers/HydratorProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Choredo\Providers;
+
+use Choredo\Hydrators\Family;
+use League\Container\ServiceProvider\AbstractServiceProvider;
+
+class HydratorProvider extends AbstractServiceProvider
+{
+
+    protected $provides = [
+        Family::class
```

Comment:
Since hydrators don't have dependencies do they need a provider? Can't the container instantiate them without it?

Reply:
It'd complain that the class isn't managed by the container :(